data/templates/success/config.yml

```diff
@@ -0,0 +1,7 @@
+name: Success Kid
+default:
+-
+link: http://knowyourmeme.com/memes/success-kid-i-hate-sandcastles
+aliases:
+- success-kid
+- i-hate-sandcastles
```

Comment:
For some reason... the template seems a bit pixelated... 

Try this one
![maybe](http://generator-meme.com/inc/media/memes/success-kid.jpg)